brokers/vscode/broker.go

```diff
@@ -97,7 +97,7 @@ func (b *Broker) PushEvents(tun *jsonrpc.Tunnel) {
 	b.Broker.PushEvents(tun, model.BrokerStatusEventType, model.BrokerResultEventType, model.BrokerLogEventType)
 }
 
-func (b *Broker) processPlugin(meta model.PluginMeta) error {
+func (b *Broker) ProcessPlugin(meta model.PluginMeta, onlyMetadata bool) error {
```

Comment:
It would interfere a lot with #37. I would suggest waiting for its merge.

Reply:
Do you mean because you did the exact same change in PR #37 (`processPlugin` to `ProcessPlugin`) ? 

I'm OK to wait for the merge of your PR, but do you have any idea if it will be merged soon ?

Reply:
Yes, want to merge it today. Seems I finished everything and addressed all the reviews so far